crontabula/__init__.py

```diff
@@ -77,7 +77,7 @@ def date_times(
                         hour=hour,
                         minute=minute,
                     )
-                    if dt.weekday() in self.day_of_week:
+                    if _day_of_week_to_cron(dt.weekday()) in self.day_of_week:
```

Comment:
Actually this `if` can be removed, we're iterating over `self.dates()` which already checks the day of week.